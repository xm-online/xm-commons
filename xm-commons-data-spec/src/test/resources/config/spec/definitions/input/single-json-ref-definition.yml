test-items:
    - key: store/GET-GEO-ADDRESS
      inputSpec: |
          {
            "type": "object",
            "properties": {
              "uaa": {
                "$ref": "#/xmDefinition/geoAddress"
              }
            }
          }

definitions:
    - key: geoAddress
      ref: json-definitions/geoAddress.json
