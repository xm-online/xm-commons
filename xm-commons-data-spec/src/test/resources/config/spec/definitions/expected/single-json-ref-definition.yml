test-items:
    - key: store/GET-GEO-ADDRESS
      inputSpec: |
          {
            "type": "object",
            "properties": {
              "uaa": {
                "$ref": "#/xmDefinition/geoAddress"
              }
            },
            "xmDefinition": {
              "geoAddress": {
                  "type": "object",
                  "properties": {
                    "location": {
                      "lat": {
                        "type": "number"
                      },
                      "lng": {
                        "type": "number"
                      }
                    },
                    "place_id": {
                      "type": "string"
                    },
                    "utcOffset": {
                      "type": "integer"
                    },
                    "components": {
                      "type": "object"
                    }
                  }
              }
            }
          }

definitions:
    - key: geoAddress
      ref: json-definitions/geoAddress.json
